config_generator/src/main.rs

```diff
@@ -0,0 +1,191 @@
+
+// Read and parse `wrangler.toml`.
+// `wrangler.example.toml` will be read if `wrangler.toml` does not exist.
+// This function panics if `wrangler.toml` contains syntax error,
+// even when a valid `wrangler.example.toml` exists.
+fn read_exsiting_config() -> (WranglerConfig, SxgConfig) {
+    let wrangler_config = std::fs::read_to_string(CONFIG_FILE)
+        .or_else(|_| std::fs::read_to_string(CONFIG_EXAMPLE_FILE))
+        .unwrap();
+    let wrangler_config: WranglerConfig = toml::from_str(&wrangler_config).unwrap();
+    let sxg_config: SxgConfig = serde_yaml::from_str(&wrangler_config.vars.sxg_config).unwrap();
+    (wrangler_config, sxg_config)
+}
+
+fn main() -> Result<(), std::io::Error> {
+    goto_repository_root()?;
+    let (mut wrangler_config, mut sxg_config) = read_exsiting_config();
+    loop {
+        let cert = read_certificate_pem_file("credentials/cert.pem");
+        let issuer = read_certificate_pem_file("credentials/issuer.pem");
+        if cert.is_ok() && issuer.is_ok() {
+            println!("Succesffuly read certificates");
+            wrangler_config.vars.cert_pem = cert.unwrap();
+            wrangler_config.vars.issuer_pem = issuer.unwrap();
+            break;
+        }
+        if let Err(msg) = cert {
+            println!("{}", msg);
+        }
+        if let Err(msg) = issuer {
+            println!("{}", msg);
+        }
+        println!(
+            "To generate certificates, please refer to\n\
+            \thttps://github.com/google/sxg-rs/blob/main/credentials/README.md"
+        );
+        Input::<String>::new()
+            .with_prompt("Press ENTER key after you generate the certificates")
+            .allow_empty(true)
+            .interact_text()
+            .unwrap();
+    }
+    let user = get_global_uesr();
+    wrangler_config.account_id = Input::new()
+        .with_prompt("What's your Cloudflare account ID?")
+        .with_initial_text(wrangler_config.account_id)
+        .interact_text()
+        .unwrap();
+    wrangler_config.zone_id = Input::new()
+        .with_prompt("What's your Cloudflare zone ID?")
+        .with_initial_text(wrangler_config.zone_id)
+        .interact_text()
+        .unwrap();
+    sxg_config.html_host = Input::new()
+        .with_prompt("What's your domain registered in Cloudflare?")
+        .with_initial_text(sxg_config.html_host)
+        .validate_with(|s: &String| -> Result<(), url::ParseError> {
+            url::Host::parse(s)?;
+            Ok(())
+        })
+        .interact_text()
+        .unwrap();
+    sxg_config.worker_host = Input::new()
+        .with_prompt("What's the domain of your Cloudflare worker?")
+        .with_initial_text(sxg_config.worker_host)
+        .validate_with(|s: &String| -> Result<(), url::ParseError> {
+            url::Host::parse(s)?;
+            Ok(())
+        })
+        .interact_text()
+        .unwrap();
+    // TODO: Allow user to customize routes
```

Comment:
May be sufficient just to print out instructions for editing the file. This should be a rare use case, given the existing `cache-control` guards.